misc/scripts/install_vulkan_sdk_macos.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
I just noticed this. macOS uses ZSH by default, perhaps this script should be made to use ZSH, or just POSIX `sh`?

Reply:
The default user shell is `zsh`, but `bash` is always installed, so it should be OK.